components/alert/__tests__/index.test.tsx

```diff
@@ -60,97 +61,78 @@ describe('Alert', () => {
     });
   });
 
-  it('support closeIcon', () => {
-    const { container } = render(
-      <Alert
-        closable
-        closeIcon={<span>close</span>}
-        message="Warning Text Warning Text Warning TextW arning Text Warning Text Warning TextWarning Text"
-        type="warning"
-      />,
-    );
-    expect(container.firstChild).toMatchSnapshot();
-  });
-
   describe('data and aria props', () => {
-    it('sets data attributes on input', () => {
-      const { container } = render(<Alert data-test="test-id" data-id="12345" message={null} />);
-      const input = container.querySelector('.ant-alert')!;
-      expect(input.getAttribute('data-test')).toBe('test-id');
-      expect(input.getAttribute('data-id')).toBe('12345');
-    });
-
-    it('sets aria attributes on input', () => {
-      const { container } = render(<Alert aria-describedby="some-label" message={null} />);
-      const input = container.querySelector('.ant-alert')!;
-      expect(input.getAttribute('aria-describedby')).toBe('some-label');
+    it('should sets data attributes on alert when pass attributes to props', () => {
+      render(
+        <Alert data-test="test-id" data-id="12345" aria-describedby="some-label" message={null} />,
+      );
+      const alert = screen.getByRole('alert');
+      expect(alert).toHaveAttribute('data-test', 'test-id');
+      expect(alert).toHaveAttribute('data-id', '12345');
+      expect(alert).toHaveAttribute('aria-describedby', 'some-label');
     });
 
     it('sets role attribute on input', () => {
-      const { container } = render(<Alert role="status" message={null} />);
-      const input = container.querySelector('.ant-alert')!;
-      expect(input.getAttribute('role')).toBe('status');
+      render(<Alert role="status" message={null} />);
+
+      expect(screen.getByRole('status')).toBeInTheDocument();
     });
   });
 
-  it('ErrorBoundary', () => {
+  it('should show error as ErrorBoundary when children have error', () => {
     jest.spyOn(console, 'error').mockImplementation(() => undefined);
     // eslint-disable-next-line no-console
     expect(console.error).toBeCalledTimes(0);
     // @ts-expect-error
     // eslint-disable-next-line react/jsx-no-undef
     const ThrowError = () => <NotExisted />;
-    const { container } = render(
+    render(
       <ErrorBoundary>
         <ThrowError />
       </ErrorBoundary>,
     );
-    // eslint-disable-next-line jest/no-standalone-expect
-    expect(container.textContent).toContain('ReferenceError: NotExisted is not defined');
+
+    expect(screen.getByRole('alert')).toHaveTextContent(
+      'ReferenceError: NotExisted is not defined',
+    );
     // eslint-disable-next-line no-console
     (console.error as any).mockRestore();
   });
 
   it('could be used with Tooltip', async () => {
```

Comment:
This test is better to be moved to `Tooltip` because it is clearly not testing `Alert`.